ansible/files/.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```

Comment:
I think this was created by my file manager, should go away (sorry for the mess, I'll update my gitignore files to prevent it from happening again)